data-prepper-core/src/integrationTest/java/com/amazon/dataprepper/integration/EndToEndRawSpanTest.java

```diff
@@ -45,6 +45,8 @@
     private static final TypeReference<Map<String, Object>> MAP_TYPE_REFERENCE = new TypeReference<Map<String, Object>>() {};
     private static final int DATA_PREPPER_PORT_1 = 21890;
     private static final int DATA_PREPPER_PORT_2 = 21891;
+    private static final int DATA_PREPPER_MAIN_PORT = 21890;
+    private static final int DATA_PREPPER_LATEST_PORT = 21891;
```

Comment:
Might suggest clarifying these names a bit (is LATEST the most recent local build, or is it the "latest" tagged docker image?). 

Or does it even matter knowing which build is on port, just re-use PORT_1 and PORT_2?

Reply:
> Or does it even matter knowing which build is on port, just re-use PORT_1 and PORT_2?

As mentioned above, for the current testing logic it is better to distinguish the data-prepper with trace-group prepper from the one w/o that prepper. But it might evolve.